robot/resources/lib/python_keywords/acl.py

```diff
@@ -219,7 +220,14 @@ def form_bearertoken_file(
         json.dump(eacl_result, eacl_file, ensure_ascii=False, indent=4)
 
     logger.info(f"Got these extended ACL records: {eacl_result}")
-    sign_bearer(shell, wif, file_path)
+    if sign:
+        sign_bearer(
+            shell=shell,
+            wallet_path=wif,
+            eacl_rules_file_from=file_path,
```

Comment:
what was the purpose to change sign_bearer with file_to and file_from if we still use same path for both ?

Reply:
There are kind of tests where we are adding signature fields to the existing bearer token, and those tests are using bearer file. For http purposes we have to create a different file where the whole bearer token file is represented in binary, and then convert those bytes to base64 to put converted binary data to http url. 
Please let me know if it's clear.